mercury/resources.py

```diff
@@ -1,13 +1,61 @@
 from tastypie.resources import ModelResource
 from tastypie.authorization import Authorization
-from mercury.models import SimulatedData
+from mercury.models import (
+    TemperatureSensor,
+    AccelerationSensor,
+    WheelSpeedSensor,
+    SuspensionSensor,
+    FuelLevelSensor,
+)
 
+# TODO check with Dave,
 
-class SimulatedDataResource(ModelResource):
-    """This class exposes an API for the SimulatedData object with the resource name
+
+class TemperatureSensorResource(ModelResource):
+    """This class exposes an API for the TemperatureData object with the resource name
+    defined below as the 'resource_name'."""
+
+    class Meta:
+        queryset = TemperatureSensor.objects.all()
+        resource_name = "tempsensor"
```

Comment:
From a REST API perspective, defining the resource as a "Temperature Sensor" means that writing data here will create a new "sensor", not a new record of data by a sensor. We'll need to better think about the resource name.

Reply:
This comment also applies to every other resource name here in this module.